website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -52,6 +52,7 @@ var QuickSearchProject = {
         promise.then(function(result) {
             self.countDisplayed(result.data.length);
             result.data.forEach(function (node) {
+                node.attributes.title = node.attributes.title.replace(/&lt;/g, '<').replace(/&gt;/g, '>').replace(/&amp;/g, '&');
```

Comment:
Seeing it all together like this makes me think we should just have a helper function. This would also make it easier if we find this in other places to keep everything consistent. Plus you'd be able to test the function.